SmartDeviceLinkTests/DevAPISpecs/SDLStreamingAudioLifecycleManagerSpec.m

```diff
@@ -402,6 +413,126 @@ __block void (^sendNotificationForHMILevel)(SDLHMILevel hmiLevel) = ^(SDLHMILeve
             });
         });
     });
+
+    describe(@"when the manager is stopped", ^{
+        context(@"if audio not stopped", ^{
+            beforeEach(^{
+                [streamingLifecycleManager.audioStreamStateMachine setToState:SDLAudioStreamManagerStateReady fromOldState:nil callEnterTransition:NO];
+                [streamingLifecycleManager stop];
+            });
+
+            it(@"should transition to the stopped state", ^{
+                expect(streamingLifecycleManager.currentAudioStreamState).to(equal(SDLAudioStreamManagerStateStopped));
+            });
+
+            it(@"should reset the saved properties", ^{
+                expect(streamingLifecycleManager.protocol).to(beNil());
+                expect(streamingLifecycleManager.hmiLevel).to(equal(SDLHMILevelNone));
+                expect(streamingLifecycleManager.connectedVehicleMake).to(beNil());
+                OCMVerify([mockAudioStreamManager stop]);
+            });
```

Comment:
These two tests can be combined

Reply:
Yes, but for readability, I think it makes sense, especially when you have to come back a year or two later to add test cases.